aerogearcatalog/metrics-apb
# [AEROGEAR-2372] Add labels to secret and configmap

**Description**
* Remove the service instance labelling task 
* Add the service instance id and service name labels to both the secret and the configmap

**Verification steps**
1. Provision the metrics service apb from this branch
2. Verify the service instance is no longer labelled with `serviceName`
3. Verify that the `metrics` configmap is correctly labelled with `mobile:enabled`, `serviceName` and `serviceInstanceID`
4. Verify that the `metrics` secret is correctly labelled with `mobile:enabled`, `serviceName` and `serviceInstanceID`
5. Verify that the `serviceInstanceID` label matches the `externalID` of the service instance
